module-system/module-implementations/sov-evm/src/dev_signer.rs

```diff
@@ -0,0 +1,79 @@
+use ethers_core::rand::rngs::StdRng;
+use ethers_core::rand::SeedableRng;
+use reth_primitives::{
+    public_key_to_address, sign_message, Bytes as RethBytes, Transaction as RethTransaction,
+    TransactionKind, TransactionSigned, TxEip1559 as RethTxEip1559, H256,
+};
+use reth_rpc::eth::error::SignError;
+use secp256k1::{PublicKey, SecretKey};
+
+use crate::evm::{EthAddress, RawEvmTransaction};
+
+/// ETH transactions signer used in tests.
```

Comment:
Move to test module, under '\tests\' dir ?

Reply:
done in `4d9f1f0bc9`